chain/network/src/peer_manager/db_types.rs

```diff
@@ -0,0 +1,55 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use chrono::{DateTime, Utc};
+use near_network_primitives::types::{PeerInfo, ReasonForBan};
+use near_primitives::time::Clock;
+use near_primitives::utils::{from_timestamp, to_timestamp};
+
+/// Status of the known peers.
+#[derive(BorshSerialize, BorshDeserialize, Eq, PartialEq, Debug, Clone)]
+pub enum KnownPeerStatus {
```

Comment:
Can we `pub(crate)` it to make it obvious that this isn't used outside?

Reply:
@matklad It's needed.
I added the following.
```
/// `pub` is used by `test-utils` in `iter_peers_from_store`
```

Maybe, we should rewrite `iter_peers_from_store` not to export private information from this module.